src/appmixer/intelswift/core/HookUpdate/HookUpdate.js

```
@@ -0,0 +1,29 @@
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { platformType, rawData, syncId } = context.messages.in.content;
+
+        // remaining to handles tenantId and projectId
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)